portal/src/components/search/SearchModal.jsx

```diff
@@ -84,17 +84,20 @@ export const SearchModal = ({ children }) => {
             <>
               {loading && !data && <Loading size={40} small />}
 
-              {data?.length > 0
-                ? data.map(form => (
+              {data?.length > 0 ? (
+                data
+                  .filter(
```

Comment:
We should change the query (line 41) when fetching this data to limit the forms by status there. Doing that on the front end in a paginated list means that each page may not show the same number of results.

We also need to make this change in the ServicesPanel.jsx component, line 130.